tests/virt_autotest/lib/teststepapi.pm

```diff
@@ -0,0 +1,202 @@
+# SUSE's openQA tests
```

Comment:
I don't understand the purpose of the whole file and also it duplicates a lot what you already have above in other files. Please tidy up.